extensions/amp-carousel/0.1/base-carousel.js

```diff
@@ -63,11 +63,6 @@ export class BaseCarousel extends AMP.BaseElement {
   }
 
   /** @override */
-  prerenderAllowed() {
```

Comment:
The default for `prerenderAllowed` is now to return `true` so this doesn't change anything for carousel. Carousel still manage its children's prerendering and layouting by itself (by calling `doPreload` inside `prerenderCallback` or `doLayout` inside `layoutCallback`).